Team1 trangdt35 add car review page and test for consumer review module, Verify title of consumer review section

TrangDao25399 opened the pull request:
add car review page and test for consumer review module
 verify title of consumer review section

TrangDao25399 committed:
add car review page for consumer review module
 add car review test for consumer review
 verify title of consumer review section

tund-hcmue commented on demo-automation-java/src/test/java/com/example/tests/BaseTest.java:
```diff
@@ -18,27 +18,18 @@ public class BaseTest {
+            keyword = new WebKeyword(driver);
             keyword.openUrl(config.getProperty("url"));
             int implicitWait = Integer.parseInt(config.getProperty("implicitWait"));
             this.wait = new WebDriverWait(driver, implicitWait);
```
Are you apply implicit wait in here?
But I haven't seen implicit applied here, it could be "driver.manage().timeouts().implicitlyWait(implicitWait, TimeUnit.SECONDS);" insted "new WebDriverWait".

TrangDao25399 replied:
got it

TrangDao25399 committed:
- Add WriteAReviewPage
- Add ConsumerReviewPage
- Verify that the browser should navigate to Write Review page when click on "Write a Review" button
- Verify that the Dislike icon should be clickable
- Verify that the Like icon should be clickable
- Verify Navigate To Consumer Review Page

ntmtrang commented on demo-automation-java/src/test/java/com/example/pages/car_reviews_pages/CarReviewDetailPageConsumerReviewModule.java:
```diff
@@ -0,0 +1,115 @@
+        return isElementCanClickable(btnDislikeIconOfReview().get(index));
+    }
+
+
```
Remove redundant space

ntmtrang commented on demo-automation-java/src/test/java/com/example/pages/car_reviews_pages/WriteAReviewPage.java:
```diff
@@ -0,0 +1,36 @@
+    private WebElement lblHeader = keyword.findElement(By.cssSelector("div.css-1ceovnz-HeaderAndLabel>h1"));
+    private WebElement lblCarDetail = keyword.findElement(By.cssSelector("h1.css-19m85bw-StyledHeading1"));
+
+
```
Remove redundant space

ntmtrang commented on demo-automation-java/src/test/java/com/example/tests/car_reviews_tests/CarReviewDetailPageConsumerReviewModuleTest.java:
```diff
@@ -0,0 +1,94 @@
+        try {
+            keyword.click(keyword.waitForElementToBeClickable(driver.findElement(By.cssSelector("#fsrFocusFirst"))));
+        } catch (Exception e) {
+        }
```
In my opinion, I think you should write these methods in another class.

TrangDao25399 replied:
I declare this function to close pop up for temporary, when another one handle this popup this function will be cleared

TrangDao25399 committed:
Merge branch 'develop' into team1-trangdt35-car-review-comsumer-review

# Conflicts:
#	demo-automation-java/src/test/java/com/example/core/keyword/WebKeyword.java
#	demo-automation-java/src/test/java/com/example/pages/BasePage.java
#	demo-automation-java/src/test/java/com/example/tests/BaseTest.java

TrangDao25399 committed:
Resolve conflict file

TrangDao25399 committed:
Merge branch 'develop' into team1-trangdt35-car-review-comsumer-review

# Conflicts:
#	demo-automation-java/src/test/java/com/example/core/reports/Test-Automation-Report.html
#	demo-automation-java/src/test/java/com/example/tests/BaseTest.java
